integrations/kube-agent-updater/cmd/teleport-kube-agent-updater/main.go

```diff
@@ -61,12 +56,22 @@ func main() {
 	var metricsAddr string
 	var probeAddr string
 	var syncPeriod time.Duration
+	var baseImageName string
+	var versionServer string
+	var versionChannel string
+	var insecureNoVerify bool
+	var disableLeaderElection bool
 
 	flag.StringVar(&agentName, "agent-name", "", "The name of the agent that should be updated. This is mandatory.")
 	flag.StringVar(&agentNamespace, "agent-namespace", "", "The namespace of the agent that should be updated. This is mandatory.")
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "healthz-addr", ":8081", "The address the probe endpoint binds to.")
 	flag.DurationVar(&syncPeriod, "sync-period", 10*time.Hour, "Operator sync period (format: https://pkg.go.dev/time#ParseDuration)")
+	flag.BoolVar(&insecureNoVerify, "insecure-no-verify-image", false, "Disable image signature verification.")
+	flag.BoolVar(&disableLeaderElection, "disable-leader-election", false, "Disable leader election, used to run the kube-agent-updater out of Kubernetes.")
+	flag.StringVar(&versionServer, "version-server", "https://update.gravitational.io/v1/", "URL of the HTTP server advertising target version and critical maintenances.")
```

Comment:
This is a bit of nit, but can you describe if the var needs the ending `/`?
Or ensure the logic below strips it in case it's there?

Currently, using at the defaults it will build a versionServerUrl with double '/'
`versionServer + "/" + versionChannel`